packages/client/hmi-client/src/components/Header.vue

```diff
@@ -40,8 +57,27 @@ defineProps<{
 			>
 				<i class="pi pi-search" />
 			</Button>
+			<Button
+				class="p-button p-button-icon-only p-button-rounded p-button-sm"
+				id="user-button"
+				@click="showUserMenu"
+			>
+				EL
```

Comment:
Unless you are planning to address this right away, I would replace this with a logout icon because nobody knows what `EL` means.

Reply:
I mean a use/profile icon.